lfs/config.go

```diff
@@ -134,11 +141,10 @@ func (c *Configuration) BatchTransfer() bool {
 		}
 
 		// Any numeric value except 0 is considered true
-		if n, err := strconv.Atoi(v); err == nil && n != 0 {
-			return true
-		}
+		n, _ := strconv.Atoi(v)
+		return n != 0
```

Comment:
It is documented, but it's documented under ParseInt, which Atoi calls. However, the value depends on the type of error. If it can't parse it as a number it returns 0, but if the number is too big to be represented it'll be the max number. Due to that, I agree with the inversion.